cloud/main.js

```diff
@@ -63,6 +63,23 @@ Parse.Cloud.define("registerNewUser", function(request, response) {
     });
 });
 
+Parse.Cloud.define("validateAlias", function(request, response) {
+    var requiredParams = ["aliasId"];
+    var params = request.params;
+    checkMissingParams(params, requiredParams, response);
+
+    var Alias = Parse.Object.extend("Alias");
+    var query = new Parse.Query(Alias);
+    query.get(params.aliasId, {
+        success: function(alias) {
```

Comment:
Why don't we just return the alias here? It's more useful than a true or false and honestly, if you're validating an alias ID, wouldn't you want to use the alias after you validate it? This is assuming you already have some handle to on the alias but maybe that's not a good assumption to make of the clients